spec/views/providers/contact.html.erb_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+
+RSpec.describe "providers/contact.html.erb" do
+  before do
+    assign(:provider, build(:provider))
+  end
+
+  context "when not an admin" do
+    before do
```

Comment:
We should make a helper for this. We've got a few different ways of doing it now and it's in a lot of controller and view specs. I'll put a card on my ideas board though, let's not worry about it in this PR.